docs/source/conf.py

```diff
@@ -87,7 +94,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "TraitsUI 4 User Manual"
+html_title = "TraitsUI 5 User Manual"
```

Comment:
If by the version information you mean the value you got from https://github.com/enthought/traitsui/blob/issue_260_docs/docs/source/conf.py#L48 then yes.